MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/MainActivity.java

```diff
@@ -573,6 +574,12 @@ private void listItems(int id) {
           new Intent(MainActivity.this, BuildingOutlineActivity.class),
           R.string.activity_query_building_outline_url, false, BuildConfig.MIN_SDK_VERSION));
 
+        exampleItemModels.add(new ExampleItemModel(
+          R.string.activity_query_highlighted_line_title,
+          R.string.activity_query_highlighted_line_description,
+          new Intent(MainActivity.this, HighlightedLineActivity.class),
+          R.string.activity_query_highlighted_line_url, false, BuildConfig.MIN_SDK_VERSION));
```

Comment:
should you also do true instead of false?

Reply:
Yep! Switched ✅